frontend/templates/views/partials/author.j2

```diff
@@ -16,7 +29,48 @@
           style="max-width: 38px"
           layout="responsive">
         </amp-img>
-        <span><strong>Written by <a href="https://github.com/{{ doc.author }}" rel="nofollow" target="_blank">@{{ doc.author }}</a></strong>{% if doc.contributors %}<br><small>with contributions from {% for contributor in doc.contributors %}{% if loop.last and not loop.first %}and {% endif %}<strong><a href="https://github.com/{{ contributor }}" rel="nofollow" target="_blank">{{ contributor }}</a></strong>{% if loop.last %}.{% else %}, {% endif %}{% endfor %}</small>{% endif %}</span>
+        {% endif %}
+
+        <span>
+          <strong>Written by 
+          {% if doc.author is iterable and doc.author is not string %}
+          {% for author in doc.author %}
+            <a href="https://github.com/{{ author }}" rel="nofollow" target="_blank">
+              @{{ author }}
+            </a>
+          {% if doc.author|length == 2 and loop.first %}
```

Comment:
Why hardcode 2 authors?

Reply:
Trying a few options, I decided to treat >1 authors as follows:

- 2 authors: "bob and judy"
- >2 authors: "bob, judy, jose"

Of course I'm not married to this idea... it just appealed to me, personally.
I made the collaborators code the same for consistency.

Reply:
ah ok, I would've expected: `bob, judy and jose`.